flow/shared/enc.go

```diff
@@ -34,6 +32,8 @@ func (e PeerDBEncKeys) Get(id string) (*PeerDBEncKey, error) {
 	return nil, fmt.Errorf("failed to find encryption key - %s", id)
 }
 
+const nonceSize = 12
```

Comment:
@heavycrystal do you think we need 24 for XChaCha?

Reply:
hould use cipher.NewX and chacha20poly1305.NonceSizeX

Reply:
granted XChaCha might be overkill

Reply:
overkill is okay